backend/spec/models/public_schools_analysis_spec.rb

```diff
@@ -58,20 +58,11 @@
     expect(project.public_schools_analysis.sca_projects[0]['subdistrict']).to eq(1)
   end
 
-  it "sets future_enrollment_multipliers correctly" do
-    expect(project.public_schools_analysis.future_enrollment_multipliers[0]['district']).to eq(15)
-    expect(project.public_schools_analysis.future_enrollment_multipliers[0]['subdistrict']).to eq(1)
-  end
-
   it "sets hs_projections correctly" do
     borough = project.public_schools_analysis.hs_projections.map {|n| n['borough']}
     expect(borough).to eq(['brooklyn'])
   end
 
-  it "sets future_enrollment_projections correctly" do
-    expect(project.public_schools_analysis.future_enrollment_projections[0]['district']).to eq(15)
-  end
```

Comment:
I don't think you should remove this test. We are not removing `future_enrollment_projections` from the model, we are only removing `future_enrollment_multipliers`.

Reply:
You're totally right - thanks for catching, fixed this in ee9e318103155f3d21232f63be0f37f3f78a9c71